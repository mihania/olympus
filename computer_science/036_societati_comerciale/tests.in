3
5 500000
10000 10000 10000 10000 10000
5 500000
100000 50000 300000 40000 100020
10 5000000
500000 500000 500000 500000 500000 500000 500000 500000 500000 500001
