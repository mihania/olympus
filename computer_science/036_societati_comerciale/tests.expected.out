0 50000
3 500020
10 5000001
